roles/update-access/test/inventory

```diff
@@ -0,0 +1,14 @@
+# Staging inventory file
+[all:vars]
+
+# create encrypted password with:
+# python -c 'import crypt; print crypt.crypt("password", crypt.mksalt(crypt.METHOD_SHA512))'
+
+user_name: user1
+user_name_password: $6$uli6pVSIQvDhxX/2$4vOEE0Jvum6OrNwoU78.wqa29m/je9ub3HXUpL6nqYxDjsqRyEAFSdbDAKxkCd0mgE9p2F/kRAPzAwFI5u8wQ1
```

Comment:
Please include a comment with what this encrypted password is in clear text so we can test the execution successfully